### ISSUE TITLE
Troublesome Source persistent after 3GC calibration

### ISSUE THREAD

**Issue by dianaklutse** [ratt-ru/CubiCal#370]
I have been battling with a troublesome source  in my image. Below is the 1GC image (first image) and the 3GC image (second image)  produced with cubical. Attached is the cubical log file, the casa list observation file and the sky model. The troublesome source is not a point source. The original data set is made up of 4096 channels but it's been down sampled to 128 channels.

![J2215](https://user-images.githubusercontent.com/22594574/80289897-059d2f00-8742-11ea-9d4b-2510a4ef87a7.png)
![wsclean-image](https://user-images.githubusercontent.com/22594574/80289903-0df56a00-8742-11ea-80c1-1e058c743422.png)

[J2215_cubical22.log](https://github.com/ratt-ru/CubiCal/files/4533929/J2215_cubical22.log)
[J2215_list_observation.log](https://github.com/ratt-ru/CubiCal/files/4533931/J2215_list_observation.log)

[testing3.lsm.html.tar.gz](https://github.com/ratt-ru/CubiCal/files/4534009/testing3.lsm.html.tar.gz)

**Comment by IanHeywood** [comment]
Hi @dianaklutse. I remember this pest from your poster!

Here are some thoughts on it:

1) I haven't done the calculation, but bandwidth smearing might be starting to bite if you average the entire MeerKAT band down to 128 channels, which will make the accuracy of models of off-centre sources derived from images suffer.

2) The attached LSM has nothing in the spectral index (spi) column, which means your sky model is entirely spectrally flat. Aside from the intrinsic (astrophysical, probably synchrotron) spectrum that the troublemaker has, it will have strong apparent spectral behaviour induced by its position in the antenna primary beam, as will every other sources away from the pointing centre. You will need to model the spectrum of the strong source more accurately to get rid of it.

3) On that note I notice your frequency solution intervals for both the G and dE terms are set to 128, so the entire band. I would suggest chunking the band up here, once the improved sky model is in place. (The dE solution 

4) The LSM also has a lot of components around the troublemaker, as well as clusters of components around other bright features. I don't have the map but I suspect these are artefacts that the source finder has picked up and rolled into the model.

5) I think in the current setup your skymodel only consists of the dE-tagged sources, so the rest of the sky won't be included while solving.

In short, I think you are asking CubiCal to calibrate your visibility data against a model that does not resemble the sky as MeerKAT sees it. It sounds like a lot but there are certainly ways to address the above, although it might mean backing up a few steps. I don't think it's necessarily a CubiCal issue though, so feel free to contact me directly if this isn't the place to discuss it.

Cheers.

**Comment by dianaklutse** [comment]
Thanks Ian, I will work on the  sky model and keep you posted.
